Jenkinsfile

```diff
@@ -49,6 +49,11 @@ podTemplate(
             stage('Install all dependencies') {
                 sh("poetry install")
             }
+            stage('Build Docs') {
```

Comment:
I think we should run tests before we build the docs. Failed tests are more interesting :)